packages/xstate-wallet/src/channel-wallet.ts

```diff
@@ -308,6 +313,10 @@ export class ChannelWallet {
     return this.store.richObjectives;
   }
 
+  public async destroy(): Promise<void> {
```

Comment:
I would prefer test-specific functions to be defined on a `TestChannelWallet` class that inherits from `ChannelWallet`.

I think it's fine to avoid that, but IMO we should clearly mark such code as test-specific, so it's easier to notice when we don't want to ship test code in production.

Reply:
In theory, `destroy` might be a functionality used for more than testing. We can imagine an mega-wallet that manages many wallet instances. Pretty far fetched at the moment...

Reply:
hmm. I'm wary of adding "just in case" methods to the public API when they were included specifically for tests.

Reply:
In general, I agree. In this case, we would need a test channel wallet + a test chain watcher. Seems like a lot of wrappers for a small destroy method.

Reply:
What about the alternative of clearly marking it as test code?

Reply:
Good idea ce2cafc2f